### PR TITLE
Update events

### PR DESCRIPTION
**Pull request by pwalczysko**
Adding several events as a post-hoc action, and 2 events as future actions.

@joshmoore @will-moore Please note that in the I2K workshop event, I have opted to shortly call the workshop ``ome-zarr``. If you want to call it (very short name only please) otherwise, feel free to correct here. Also, the event is highlighted as "OME workshops". 
Possibly you have other suggestion for that as well, feel free to correct. Recommending to use find and replace function for the "ome-zarr", as it is in 2 different files on several places.

Built at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

Note: also removing the red banner about outage in this PR - this banner never made it to op...py.org website, and imho we have handled the outage already.

cc @sbesson

### CODE REVIEW

**Comment by pwalczysko** [comment]
Built at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

**Comment by joshmoore** [comment]
Deployed at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ (You beat me to it!)

Screen Shot 2020-11-25 at 19 38 56

I think "ome-zarr" will not work, since it does not do justice to Tischi's contributions. Options:
 - "Next-generation file formats"
 - "Next-generation file formats (NGFF)"
 - ... 

NB: I notice that the dividing line has somehow gone missing.

**Comment by pwalczysko** [comment]
> dividing line has somehow gone missing.

Not sure what you mean with dividing line.

**Comment by joshmoore** [comment]
> Not sure what you mean by `dividing line`. Is that a real line in the html or some thought concept ?

If you look at the entire events page, you will see dividers between the older entries. This is missing in the newer ones.

**Comment by pwalczysko** [comment]
``Four 4-hour demos/workshops on how to use ome-zarr and OMERO held remotely.`` -> ``Four 4-hour demos/workshops on Next-generation file formats and OMERO held remotely.``

How is that @joshmoore ? (the Next-generation ... will be in inverted commas I imagine)
Also, please note the "OME Workshops" - first in your screenshot - is that okay ?

**Comment by joshmoore** [comment]
> Also, please note the "OME Workshops" - first in your screenshot - is that okay ?

I think that's fine.

> Four 4-hour demos/workshops on Next-generation file formats and OMERO held remotely.

:+1:

**Comment by pwalczysko** [comment]
@sbesson Not sure why https://snoopycrimecop.github.io/www.openmicroscopy.org/ does not show the results of the last 3 commits - can I cause a rebuild manually somehow please ?

**Comment by pwalczysko** [comment]
Updated the header Re: outage banner.

**Comment by joshmoore** [comment]
@pwalczysko : I launched https://merge-ci.openmicroscopy.org/jenkins/job/WEBSITE-push/111/

**Comment by pwalczysko** [comment]
Thanks @joshmoore . Indeed, https://snoopycrimecop.github.io/www.openmicroscopy.org/ looks as it should look (all commits included). Ready for review.

**Comment by will-moore** [comment]
By tomorrow you'll need another fix to remove the 'future' star from one workshop.
Is it worth just removing both the stars, since highlighting that they are future (but you're already too late to apply) is probably not worth the effort?

**Comment by pwalczysko** [comment]
Thanks @will-moore , fix pushed and rebuilt https://snoopycrimecop.github.io/www.openmicroscopy.org/

File: events/index.html [hunk 1]
```diff
@@ -9,10 +9,34 @@
...
+                <h6><i class="fa fa-calendar"></i> November 18, 2020</h6>
+                <h2><a href="omero-users-workshop-eastbio-edinburgh-online-november-2020.html">OMERO Workshop, EastBio, held remotely</a></h2>
+                <p class="card-caption">A 45 minutes demo/workshop on how to use OMERO.</p>
+            </div>
+            <hr class="medium-8">
             <div class="small-12 medium-8 medium-offset-2 columns">
                 <h6><i class="fa fa-calendar"></i> May 26-28, 2020</h6>
                 <h2><a href="{{ site.baseurl }}/events/ome-community-meeting-2020/">2020 OME Community Meeting</a></h2>
-                <p class="card-caption">The 2020 OME Community Meeting will be held remotely from May 26th to May 28th 2020.</p>
+                <p class="card-caption">The fdsafdsfa 2020 OME Community Meeting will be held remotely from May 26th to May 28th 2020.</p>
```
Comment (joshmoore) [inline events/index.html:39]:
```suggestion
                <p class="card-caption">The 2020 OME Community Meeting will be held remotely from May 26th to May 28th 2020.</p>
```
Can we remove this? ;)

Comment (pwalczysko) [reply events/index.html]: ufff, great catch, thanks @joshmoore

Comment (pwalczysko) [reply events/index.html]: the local docker command via jekyll somehow stopped to rebuild when the html is changed and saved. I did not understand it first, and was injecting some nonsense into the text to make sure I will see any change in the built , and then forgot to remove it.

**Comment by pwalczysko** [comment]
Thanks @joshmoore - rebuilt again with your commit on https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

**Comment by joshmoore** [comment]
LGTM. Merging and tagging (after checking for other open PRs)

**Pull request merged**